### PR TITLE
dictd

### PR DESCRIPTION
**Pull request by ayatakesi**
I create libmaa and dictd repository.

### CODE REVIEW

File: packages/dictd/dictd.patch [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+--- ./parse.c	2018-02-07 15:32:52.416533087 +0900
++++ ../dictd-1.12.1.orig/parse.c	2008-12-08 01:50:05.000000000 +0900
+@@ -90,10 +90,10 @@
+    char              *buffer;
+    const char        **pt;
+    static const char *cpp = NULL;
+-   static const char *cpps[] = { "@TERMUX_PREFIX@/lib/cpp",
+-                                 "@TERMUX_PREFIX@/usr/lib/cpp",
+-                                 "@TERMUX_PREFIX@/usr/ccs/lib/cpp",	/* Solaris */
+-                                 "@TERMUX_PREFIX@/usr/lang/cpp",
```
Comment (Grimler91) [inline packages/dictd/dictd.patch]: one /usr/ too many here, TERMUX_PREFIX=/data/data/com.termux/files/usr

Comment (ayatakesi) [reply packages/dictd/dictd.patch]: Thanks, I will fix it.

**Comment by Grimler91** [comment]
Looks like you need to override ld so that it uses $(LD) instead of /usr/bin/ld

**Comment by ayatakesi** [comment]
Ok,I will do it. Thanks!

File: packages/dictd/build.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=http://sourceforge.net/projects/dict/
+TERMUX_PKG_DESCRIPTION="Online dictionary client and server"
+TERMUX_PKG_VERSION=1.12.1
+TERMUX_PKG_SRCURL=https://downloads.sourceforge.net/project/dict/dictd/dictd-${TERMUX_PKG_VERSION}/dictd-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=a237f6ecdc854ab10de5145ed42eaa2d9b6d51ffdc495f7daee59b05cc363656
+TERMUX_PKG_DEPENDS="libmaa"
+TERMUX_PKG_BUILD_DEPENDS="libmaa-dev"
+
+termux_step_pre_configure() {
+	LDLAGS+=" -lmaa"
```
Comment (stephengroat) [inline packages/dictd/build.sh]: is this being used? i know you meant `LDFLAGS`, but could this entire step be deleted?

Comment (ayatakesi) [reply packages/dictd/build.sh]: Ok, I will try to delete these steps.
Thanks!

**Pull request merged**